geode-core/src/main/java/org/apache/geode/management/internal/configuration/realizers/ConfigurationRealizer.java

```diff
@@ -29,15 +29,25 @@
  */
 @Experimental
 public interface ConfigurationRealizer<T extends AbstractConfiguration<R>, R extends RuntimeInfo> {
-  RealizationResult create(T config, InternalCache cache);
+  default RealizationResult create(T config, InternalCache cache) {
```

Comment:
Having most all of the methods on an interface have default implementations that throw an exception seems like an anti-pattern. The default impl of isReadOnly makes sense but I think the old create/update/delete was better when each implementer of the interface was forced to deal with how they would implement those methods. Now it seems like it is really easy to implement ConfigurationRealizer but it will blow up at runtime.

Reply:
used a ReadonlyRealizer to do this.